betfair/models.py

```diff
@@ -138,7 +138,7 @@ class Match(BetfairModel):
 
 
 class Runner(BetfairModel):
-    selection_id = Field(DataType(float), required=True)
+    selection_id = Field(DataType(six.text_type), required=True)
```

Comment:
I never had any problems with this. Using python3.4 on OS X and Linux. Float type works as expected.